MapboxNavigation/NavigationViewController.swift

```diff
@@ -687,6 +688,17 @@ extension NavigationViewController: NavigationServiceDelegate {
         return navigationComponents.allSatisfy { $0.navigationServiceShouldDisableBatteryMonitoring(service) }
     }
     
+    public func locationManagerDidChangeAuthorization(_ manager: CLLocationManager) {
+        let title = NSLocalizedString("ENABLE_PRECISE_LOCATION", bundle: .mapboxNavigation, value: "Enable precise location to  navigate", comment: "Label indicating precise location is off and needs to be turned on to navigate")
```

Comment:
This is a new user-facing string. You’ve already followed most of [the steps for adding a new user-facing string](https://github.com/mapbox/mapbox-navigation-ios/blob/64dd90e79e882516d50e3577d854f80bbc1da2b8/CONTRIBUTING.md#adding-user-facing-text), but the base localization file still needs to be updated using a script.